### PR TITLE
FWSP1-120 Minor backend improvements

### PR DESCRIPTION
**Pull request by googletorp**

### CODE REVIEW

File: web/themes/custom/kk_admin/src_assets/scss/components/config.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Hide compact link on config page.
```
Comment (xendk) [inline web/themes/custom/kk_admin/src_assets/scss/components/config.scss:1]: What's  a compact link?

File: web/themes/custom/kk_admin/src_assets/scss/components/menu.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.menu-edit-form {
```
Comment (xendk) [inline web/themes/custom/kk_admin/src_assets/scss/components/menu.scss:1]: Ah... Wouldn't `['#access'] = false;` be prettier? Especially as I suspect this also hides it for user 0.

File: web/themes/custom/kk_admin/src_assets/scss/components/config.scss (hunk 1 above)
Comment (googletorp) [reply web/themes/custom/kk_admin/src_assets/scss/components/config.scss:1]: It's a link that compacts the configuration page (hides the description texts)

File: web/themes/custom/kk_admin/src_assets/scss/components/menu.scss (hunk 2 above)
Comment (googletorp) [reply web/themes/custom/kk_admin/src_assets/scss/components/menu.scss:1]: #access false will also hide it for all users, but #access $user->access('no permission) will not hide it for user 1. As we only visually hide it, we can still edit it if we want to, so seemed easier / better

File: web/themes/custom/kk_admin/src_assets/scss/components/config.scss (hunk 1 above)
Comment (xendk) [reply web/themes/custom/kk_admin/src_assets/scss/components/config.scss:1]: Would be nice with a comment about *why*.

**Pull request merged**